code/drasil-example/Drasil/DblPendulum/References.hs

```diff
@@ -0,0 +1,17 @@
+module Drasil.DblPendulum.References where
+
+import Language.Drasil
+import Data.Drasil.Citations (cartesianWiki, accelerationWiki, velocityWiki, parnasClements1986)
+import Data.Drasil.People (rcHibbeler)
+
+citations :: BibRef
+citations = [accelerationWiki, velocityWiki, hibbeler2004, cartesianWiki, parnasClements1986]
+
+hibbeler2004 :: Citation
```

Comment:
This is not good - it represents duplication of the information that is in `Pendulum`. It was better the way it was before. Maybe take the opportunity to move this to `data-drasil`?